src/stores/SpaceStore.tsx

```diff
@@ -467,6 +490,15 @@ export class SpaceStoreClass extends AsyncStoreWithClient<IState> {
             case "view_room": {
                 const room = this.matrixClient?.getRoom(payload.room_id);
 
+                // persist last viewed room from a space
+
+                // We don't want to save if the room is a
+                // space room since it can cause problems
```

Comment:
whats the problem is can cause?

Reply:
This is weird - I remember there being a problem but I can't repro it now. If any issues arise we know where to look